integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -94,7 +99,12 @@ def get_config_from_frequency_sweep(full_df):
                          '; run a frequency sweep before using this analysis.\n')
         sys.exit(1)
 
-    output = get_config_from_frequency_sweep(df)
+    if args.gpu_energy_margin < 0:
+        sys.stderr.write('<geopm> Error: GPU energy margin must be non-negative\n')
```

Comment:
When printing warnings, I've seen we place the "<geom>" tag after the "Warning" header/label. I do see that for printing errors, the pattern is to place it before. Is there a reason why we place the "<geopm>" differently for errors and warnings?

Reply:
From review: Error/Warning: <geopm> script_name: Actual message"